data/default_os_images.json

```diff
@@ -70,5 +70,29 @@
         "cpu_architecture": "ppc64le",
         "url": "https://mirror.openshift.com/pub/openshift-v4/ppc64le/dependencies/rhcos/4.12/4.12.0/rhcos-4.12.0-ppc64le-live.ppc64le.iso",
         "version": "412.86.202212081411-0"
+    },
+    {
+        "openshift_version": "4.13",
+        "cpu_architecture": "x86_64",
+        "url": "https://mirror.openshift.com/pub/openshift-v4/x86_64/dependencies/rhcos/4.12/4.12.0/rhcos-4.12.0-x86_64-live.x86_64.iso",
+        "version": "412.86.202212081411-0"
+    },
+    {
+        "openshift_version": "4.13",
+        "cpu_architecture": "arm64",
+        "url": "https://mirror.openshift.com/pub/openshift-v4/aarch64/dependencies/rhcos/4.12/4.12.0/rhcos-4.12.0-aarch64-live.aarch64.iso",
+        "version": "412.86.202212081411-0"
+    },
+    {
+        "openshift_version": "4.13",
+        "cpu_architecture": "s390x",
+        "url": "https://mirror.openshift.com/pub/openshift-v4/s390x/dependencies/rhcos/4.12/4.12.0/rhcos-4.12.0-s390x-live.s390x.iso",
+        "version": "412.86.202212081411-0"
+    },
+    {
+        "openshift_version": "4.13",
+        "cpu_architecture": "ppc64le",
+        "url": "https://mirror.openshift.com/pub/openshift-v4/ppc64le/dependencies/rhcos/4.12/4.12.0/rhcos-4.12.0-ppc64le-live.ppc64le.iso",
+        "version": "412.86.202212081411-0"
     }
```

Comment:
maybe we don't need those (not yet requirement to test 4.13 with PowerPC / zSystems)

Reply:
Done